src/submission/migration/migration-entities.ts

```diff
@@ -62,9 +62,5 @@ export type NewSchemaVerificationResult = {
 };
 
 export type DonorMigrationSchemaErrors = Array<
-  {
-    [clinicalAlias in EntityAlias | string]: ReadonlyArray<
-      dictionaryEntities.SchemaValidationError
-    >;
-  }
+  [ClinicalEntitySchemaNames, ReadonlyArray<dictionaryEntities.SchemaValidationError>]
```

Comment:
Changing from an object to an array here is giving me questions

Reply:
The object being used was basically using an object with a single key/ value pair as a tuple, so i just made the whole thing into a tuple. This reduces complexity when using this data... for example we don't need to use `Object.entries` to access the key/value pair, and don't need to assume that only one key/value pair exists in the object... all that is baked into the tuple.

Reply:
That's great, had not considered that
Usages of DonorMigrationSchemaErrors will have to be updated down stream

Reply:
Are there other usages? looked like it was only used in one place. TS still compiles so I think we're safe. Do you know of places this could impact?

Reply:
Several unit tests fail as a result of this change, to me that indicates there could be additional impacts
https://jenkins.qa.cancercollaboratory.org/blue/organizations/jenkins/ARGO%2Fargo-clinical/detail/fix%2Fdata-query-large-filters/3/pipeline/